cram/encoding/ltf8/ltf.go

```diff
@@ -0,0 +1,170 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package ltf8 provides LTF-8 integer encoding.
+//
+// LTF-8 integer encoding is specified in the CRAM format specification
+// section 2.3.
+package ltf8
+
+var pop = [16]byte{
+	0:  8,
+	1:  7,
+	4:  3,
+	5:  6,
+	6:  1,
+	9:  4,
+	10: 2,
+	11: 5,
+	14: 0,
+}
+
+// nlo returns the number of leading set bits in x.
+func nlo(x byte) int {
```

Comment:
`nlo(x)` (and pop) can be replace by `bits.LeadingZeros8(^x)`

Reply:
I guess `bits` dependency would truncate support to more recent versions of go.

Reply:
I'm happy to drop that. "math/bits" didn't exist when I wrote this.

Reply:
Done